release_notes/v1.5.5.md

```diff
@@ -0,0 +1,49 @@
+v1.5.4 Release Notes - July 8, 2022
+===================================
+
+v1.5.4 is a maintenance release, providing updates for the following issues in the Fabric CA: 
+
+- Closes [Issue #305](https://github.com/hyperledger/fabric-ca/issues/305) : Issue with re-enrolling certificates 
+
+- Closes [Issue #309](https://github.com/hyperledger/fabric-ca/issues/309) : Interoperability tests are failing with latest CA updates
```

Comment:
We don't need to mention this one, since the break was introduced after v1.5.4. That is, it will not impact users of the official releases and therefore does not need to be mentioned in the release notes as a release-to-release change.